pkg/network/setup/network.go

```diff
@@ -97,3 +120,16 @@ func (n *VMNetworkConfigurator) SetupPodNetworkPhase2(domain *api.Domain) error
 	}
 	return nil
 }
+
+func isIfaceAlreadyAvailableInVM(ifacesToHotplug map[string]struct{}, networkName string) bool {
```

Comment:
I don't understand how this proves the interface is already in the VM.
What do you mean by already in the VM, in the VMI spec? Or actually in the guest?
I believe you should check it is actually in the guest/libvirt/pass phase1, I cannot undestand how this method here verifies it. I cannot think of a scenario no `networkName` from `vmi.Spec.Networks` will fit the `ifacesToHotplug`.

Reply:
This code is gone.